src/core/Akka/Actor/Settings.cs

```diff
@@ -309,7 +309,7 @@ private static string GetProviderClass(string provider)
         /// <inheritdoc/>
         public override string ToString()
         {
-            return Config.ToString();
+            return Config.Root.ToString();
```

Comment:
Is this going to skip fallback values during debugging?

Reply:
No, `Root` is a cumulative merged object, it already includes all of the fallbacks in the `Config`.